src/borg/helpers.py

```diff
@@ -1247,6 +1234,46 @@ def __del__(self):
             self.logger.removeHandler(self.handler)
             self.handler.close()
 
+
+def justify_to_terminal_size(message):
+    terminal_space = get_terminal_size(fallback=(-1, -1))[0]
+    # no need to justify if we're not outputing to a terminal
```

Comment:
typo and also maybe invert the "not"s:

need to justify if we are outputting to a terminal